### ISSUE TITLE
Extra characters \ufeff appear when unchecking styles on an empty cursor

### ISSUE THREAD

**Issue by fredknyazev** [tinymce/tinymce#5352]
Extra characters \ufeff appear when unchecking styles on an empty cursor

Steps:
1. Type character in TinyMce
2. Select char and select styles: Bold, Italic, Strickethgrough
3. Click to end or press End
4. Unselect all selected styles: Bold, Italic, Strickethgrough
5. Type character
<-----Issue
6. Try moving cursor Left Arrow to the beginning of line

The cursor is not expected to get stuck

Browser: Chrome, FF

Repeatable on https://www.tiny.cloud/
![image](https://user-images.githubusercontent.com/53004232/72593709-b8294e80-3916-11ea-8058-007e3bf8b0a7.png)

**Comment by lostkeys** [comment]
Do the character show up as an HTML entity in the source code when using the [code plugin](https://www.tiny.cloud/docs/plugins/code/) (meaning like &nbsp; etc)?
Also you say the cursor "gets stuck", how do you mean stuck?

**Comment by fredknyazev** [comment]
No, the symbol is not visible visually. As I understand it, it is created when we assign a style to the cursor even without text. And no, we do not use plugins.
By “cursor gets stuck” I mean that when we move the cursor using the left arrows, it visually stays in place, just passing the symbol \ufeff.
But if we left the cursor through these invisible characters, they are erased and there are no more problems.
Sorry for my English, I'm from Russia, I speak via Google Translate

**Comment by lostkeys** [comment]
Ok, apparently it's used to control the cursor position, but having multiple ones is a bug. What browsers do you experience this issue with?

**Comment by fredknyazev** [comment]
Chrome 79, FF 72, IE11, Edge 42
